package.json

```diff
@@ -18,7 +18,7 @@
     "pretypecheck": "npm run buildEnvTypes",
     "typecheck-main": "tsc --noEmit -p packages/main/tsconfig.json",
     "typecheck-preload": "tsc --noEmit -p packages/preload/tsconfig.json",
-    "typecheck-renderer": "tsc --noEmit -p packages/renderer/tsconfig.json",
+    "typecheck-renderer": "cd packages/renderer && vue-tsc --noEmit",
```

Comment:
You could use `vue-tsc --noEmit -p packages/renderer/tsconfig.json`.

Reply:
Hmm... I've tried this, but it doesn't seem to work.
```
vue-tsc --noEmit -p packages/renderer/tsconfig.json
```
Not founded error. But 
```
cd packages/renderer && vue-tsc --noEmit
```
does.

I just added a [commit with an error](https://github.com/cawa-93/vite-electron-builder/pull/162/commits/e1e4e7daf70209e6413df955baddc686fa9a9a64#diff-fc9f137a66dd3adaed4151ff937426b60f925a59179b4057cf540b8ea9f572abR24). But the type check was [successful](https://github.com/cawa-93/vite-electron-builder/pull/162/checks?check_run_id=2122363220#step:8:6).

Reply:
Sorry it's a bug and I fixed it: https://github.com/johnsoncodehk/vue-tsc/commit/e0d8a363aa4cce30dca47b8ee540a85190af8365

Not yet publish to 0.0.14, please use old script for now.